File: 26-Remove Duplicates from Sorted Array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if not nums:
            return 0
        
        j = 0 
        for i in range(len(nums)-1):
            if nums[i] < nums[i+1]:
                nums[j] = nums[i]
                j+=1
        nums[j] = nums[-1]
        
        return j+1

File: JavaScript/26-Remove Duplicates from Sorted Array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    if (!nums.length) return 0;
    
    let j = 0;
    
    for (let i=0; i<nums.length-1; i++){
        if (nums[i] < nums[i+1]){
            nums[j] = nums[i];
            j+=1;
        }
    }
    nums[j] = nums[nums.length-1];
    return j+1;
};